core/test/functional/routes/api/posts_spec.js

```diff
@@ -7,6 +7,23 @@ var should = require('should'),
     ghost = testUtils.startGhost,
     request;
 
+function mobiledocDocument(content) {
```

Comment:
I've duplicated this function a number of times - where would be the best place to define this as a test util?